homeassistant/components/arwn/sensor.py

```diff
@@ -135,6 +139,11 @@ def name(self):
         """Get the name of the sensor."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return self.entity_id
```

Comment:
You can't use entity ID. A unique ID can be used to change the entity ID, which will then change the unique ID again.

Reply:
ack, will use mqtt topic